generators/entity-client/templates/angular/src/main/webapp/app/entities/update/entity-management-update.component.spec.ts.ejs

```diff
@@ -124,6 +129,79 @@ _%>
     <%_ } _%>
         });
 <%_ } _%>
+<%_ if (relationships.filter(rel => rel.ownerSide).length > 0) { _%>
+
+        describe('Loading relationships', () => {
+            describe('loadRelationshipsOptions', () => {
+    <%_ for (const relationshipsByEntityNeedingOptions of allRelationshipsByEntityNeedingOptions) { _%>
+        <%_ const relationshipsWithCustomUniqueOptions = relationshipsByEntityNeedingOptions.filter(rel => rel.relationshipType === 'one-to-one' && rel.otherEntityName !== 'user'); %>
+        <%_ const relationshipsWithCustomSharedOptions = relationshipsByEntityNeedingOptions.filter(rel => !relationshipsWithCustomUniqueOptions.includes(rel)); %>
+        <%_ if (relationshipsWithCustomSharedOptions.length > 0) { _%>
+            <%_ const otherEntity = relationshipsByEntityNeedingOptions[0].otherEntity _%>
+
+                it('Should call <%= otherEntity.entityAngularName %> query and add missing value', () => {
+                    fakeAsync(() => {
```

Comment:
```suggestion
```
If I tested then this code was not executed if using `fakeAsync`. After removing `fakeAsync` test was ran and this test failed.